### PR TITLE
Added required parameter to document DOC-string to produce Documentation.

### LINKED ISSUE #64654: [WARNING]: template parsing did not produce documentation
##### ISSUE TYPE
- Bug Report

##### ANSIBLE VERSION

```paste below

```

##### CONFIGURATION

```paste below

```

##### STEPS TO REPRODUCE

```yaml

```

##### ACTUAL RESULTS

```paste below

```
$ansible-doc -l | grep UNDOCUMENTED
[WARNING]: template parsing did not produce documentation.
[WARNING]: win_template parsing did not produce documentation.
template                                                      UNDOCUMENTED                                                   
win_template                                                  UNDOCUMENTED                                                   

There're 2 modules undocumented.

$ansible --version
ansible 2.9.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/smith/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.7/site-packages/ansible-2.9.0-py3.7.egg/ansible
  executable location = /usr/bin/ansible
  python version = 3.7.5 (default, Oct 21 2019, 7:17:39) [GCC 7.4.0]

#

### PR DESCRIPTION
**Pull request by sbulage**
Signed-off-by: Satyajit Bulage <sbulage@redhat.com>

##### SUMMARY

Fixes #64654

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

template and win_template

### CODE REVIEW

**Comment by sbulage** [comment]
cc @acozine

File: lib/ansible/modules/windows/win_template.py [hunk 1]
```diff
@@ -13,6 +13,7 @@
 ---
 module: win_template
 version_added: "1.9.2"
+short_description: Create file from template or Backup given file.
```
Comment (jborean93) [inline lib/ansible/modules/windows/win_template.py]: Should just be what was on the original `short_description` in the template_common file.

File: lib/ansible/modules/files/template.py [hunk 2]
```diff
@@ -17,6 +17,7 @@
 ---
 module: template
 version_added: historical
+short_description: Create a files and directories by using templates.
```
Comment (jborean93) [inline lib/ansible/modules/files/template.py]: SHould just be what was on the original `short_description` in the template_common file.

File: lib/ansible/plugins/doc_fragments/template_common.py [hunk 3]
```diff
@@ -11,8 +11,8 @@ class ModuleDocFragment(object):
 
     # Standard template documentation fragment, use by template and win_template.
     DOCUMENTATION = r'''
-short_description: Template a file out to a remote server
 description:
+- Template a file out to a remote server.
```
Comment (jborean93) [inline lib/ansible/plugins/doc_fragments/template_common.py]: I don't think this is needed, seems like we just need to remove `short_description` here and just duplicate it across to `template.py` and `win_template.py`.

**Review by jborean93 — CHANGES_REQUESTED** [review]

**Review by jborean93 — APPROVED** [review]

**Comment by jborean93** [comment]
I've pushed the requested changes so we can merge this in and get the issue fixed.

**Comment by jborean93** [comment]
Backport PR to stable-2.9 https://github.com/ansible/ansible/pull/65518

**Pull request merged**